src/concrete/ml/quantization/quantized_module.py

```diff
@@ -691,11 +691,6 @@ def compile(
             jit=False,
         )
 
-        # CRT simulation is not supported yet
-        # TODO: https://github.com/zama-ai/concrete-ml-internal/issues/3841
-        if not USE_OLD_VL:
```

Comment:
not necessary anymore, a circuit calls `enable_fhe_simulation` automatically when running `simulate`